src/models/util/transactionUtils.ts

```diff
@@ -492,6 +492,7 @@ export class TransactionUtils {
       txnStatus = await AppState.chainAPI.transactionAPI.getTransactionStatus(hash);
       if(txnStatus.group == 'partial'){
         txn = await AppState.chainAPI.transactionAPI.getPartialTransaction(hash);
+        console.log(txn)
```

Comment:
remember to remove debugging if this is deployed to production

Reply:
Yeah @shinneng , noted